modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/PageAuditApp.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClayLayout from '@clayui/layout';
+import {ClayTooltipProvider} from '@clayui/tooltip';
+import React, {useMemo, useState} from 'react';
+
+import EmptyPageAudit from './components/EmptyPageAudit';
+
+const MOCK_DATA = {
+	assetsPath: '/o/analytics-reports-web/assets',
+	canonicalURL: 'http://localhost:8080/en/web/guest/w/basic-content-page',
+	defaultLanguage: 'en-US',
+	showButton: false,
```

Comment:
I'd be more specific and using showConfigurePageSpeedButton instead of using showButton.